lib/active_model/serializer/links.rb

```diff
@@ -0,0 +1,33 @@
+module ActiveModel
+  class Serializer
+    module Links
+      extend ActiveSupport::Concern
+
+      included do
+        with_options instance_writer: false, instance_reader: true do |serializer|
+          serializer.class_attribute :_links # @api private
+          self._links ||= {}
+        end
+
+        extend ActiveSupport::Autoload
+      end
+
+      module ClassMethods
+        def inherited(base)
+          super
+          base._links = _links.dup
+        end
+
+        # Define a link on a serializer.
+        # @example
+        #   link :self { "/posts/#{object.id}" }
```

Comment:
No missing comma (this is a block, not a hash). As far as absolute urls are concerned, yes :)